Updating document to reflect current technical state. (#9410)
MicrosoftDocs/sql-docs: azure-sql/migration-guides/modernization.md

#### What is Azure SQL?

> [Azure SQL](../azure-sql-iaas-vs-paas-what-is-overview.md) is a family of services that use the SQL Server database engine in the Azure Cloud. The following services belong to Azure SQL: [Azure SQL Database](../database/sql-database-paas-overview.md) (SQL Database), [Azure SQL Managed Instance](../managed-instance/sql-managed-instance-paas-overview.md) (SQL Managed Instance) and [SQL Server on Azure VMs](../virtual-machines/index.yml) (SQL VM).

Reviewer: In a lot of cases, we aren't allowed to abbreviate service names. This is one such example.

```suggestion
[Azure SQL](../azure-sql-iaas-vs-paas-what-is-overview.md) is a family of services that use the SQL Server database engine in the Azure Cloud. The following services belong to Azure SQL: [Azure SQL Database](../database/sql-database-paas-overview.md) (SQL Database), [Azure SQL Managed Instance](../managed-instance/sql-managed-instance-paas-overview.md) (SQL Managed Instance) and [SQL Server on Azure VMs](../virtual-machines/index.yml).
```

Author: But you are using SQL VM later in the same article. few times like in lines 45, 59, 75, 77, 91, 105, 149, 155, 159, and none of those abbreviations have been written by me. I just see it logical to add this shortened form at the beginning of the article for better understanding.

Reviewer: They shouldn't be there then, so I'll have to remove all references to that. That's part of the excitement of inheriting content.

Author: Happy to help then :)

For example, SQL Managed Instance and SQL Database (single database and elastic pool) come with built-in HA. Also, Business Critical (SQL Managed Instance) and Premium (SQL Database) tiers provide read-only replicas at no additional cost, while SQL Database Hyperscale tier allows HA and named [secondary replicas](../database/service-tier-hyperscale-replicas.md) for [read scale-out](../database/read-scale-out.md) at no license cost. Additionally, [Software Assurance](https://www.microsoft.com/licensing/licensing-programs/software-assurance-default) customers can use their on-premises SQL Server license on Azure by applying [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-benefit/) (AHB). Software Assurance also lets you implement [free passive HA and DR secondaries using SQL VM](../virtual-machines/windows/business-continuity-high-availability-disaster-recovery-hadr-overview.md#free-dr-replica-in-azure).

> In addition, every Azure SQL service provides you the option to reserve instances in advance (for 1 year or 3 years term) and obtain significant additional savings. [Dev/Test pricing plans](https://azure.microsoft.com/pricing/dev-test/#devtest) provide a way to further reduce development costs. Finally, check the following article on how you can [Optimize your Azure SQL Managed Instance cost with Microsoft Azure Well-Architected Framework](https://techcommunity.microsoft.com/t5/azure-sql-blog/optimize-your-azure-sql-managed-instance-cost-with-microsoft/ba-p/2235216).

Reviewer: ```suggestion
In addition, every Azure SQL service provides you the option to reserve instances in advance (for terms of either 1 year or 3 years) and obtain significant additional savings. [Dev/Test pricing plans](https://azure.microsoft.com/pricing/dev-test/#devtest) provide a way to further reduce development costs. Finally, check the following article on how you can [Optimize your Azure SQL Managed Instance cost with Microsoft Azure Well-Architected Framework](https://techcommunity.microsoft.com/t5/azure-sql-blog/optimize-your-azure-sql-managed-instance-cost-with-microsoft/ba-p/2235216).
```

For implementing BCDR solutions in Azure SQL, you benefit from built-in HA replicas of SQL Managed Instance and SQL Database or [free passive HA and DR secondaries using SQL VM](../virtual-machines/windows/business-continuity-high-availability-disaster-recovery-hadr-overview.md#free-dr-replica-in-azure). Also, Business Critical (SQL Managed Instance) and Premium (SQL Database) tiers provide read-only replicas at no additional cost, while SQL Database Hyperscale tier allows HA and named [secondary replicas](../database/service-tier-hyperscale-replicas.md) for [read scale-out](../database/read-scale-out.md) at no license cost. In addition, make sure to check different [Dev/Test pricing plans](https://azure.microsoft.com/pricing/dev-test/#devtest).

> If you're interested to understand how you can save up to 64% by moving to Azure SQL please check ESG (Environmental, Social and Governance Impact report) report on [The Economic Value of Migrating On-Premises SQL Server Instances to Microsoft Azure SQL Solutions](https://azure.microsoft.com/resources/the-economic-value-of-migrating-on-premises-sql-server-instances-to-microsoft-azure-sql-solutions/). Finally, check the following article on how you can [Optimize your Azure SQL Managed Instance cost with Microsoft Azure Well-Architected Framework](https://techcommunity.microsoft.com/t5/azure-sql-blog/optimize-your-azure-sql-managed-instance-cost-with-microsoft/ba-p/2235216).

Reviewer: ```suggestion
If you would like to understand how you can save up to 64% by moving to Azure SQL please check the Environmental, Social and Governance Impact (ESG) report on [The Economic Value of Migrating On-Premises SQL Server Instances to Microsoft Azure SQL Solutions](https://azure.microsoft.com/resources/the-economic-value-of-migrating-on-premises-sql-server-instances-to-microsoft-azure-sql-solutions/). Finally, check the following article on how you can [Optimize your Azure SQL Managed Instance cost with Microsoft Azure Well-Architected Framework](https://techcommunity.microsoft.com/t5/azure-sql-blog/optimize-your-azure-sql-managed-instance-cost-with-microsoft/ba-p/2235216).
```

#### What is Azure Hybrid Benefit (AHB)?

> Unique to Azure, Azure Hybrid Benefit (AHB) is a licensing benefit that allows you bringing your existing Windows Server and/or SQL Server licenses (both Standard and Enterprise) with Software Assurance (SA) to Azure IaaS & Paas services. If you are planning to bring only existing SQL Server licenses, Azure Hybrid Benefits allow you bringing licences to Azure Data Factory SQL Server Integration Services, Azure SQL Database (both Elastic Pool and Single Database), SQL Managed Instances (General Purpose, Business Critical and Hyperscale) and Azure Arc-enabled SQL Managed Instance (both General Purpose and Business Critical). AHB can bring you up to 85% savings compared to pay-as-you-go pricing in Azure SQL, when combined with reservations savings and extended security updates. Azure Hybrid Benefit is not available in the serverless compute tier of Azure SQL Database. For more detailed information, how to use AHB with your own licenses see [Microsoft Azure Services Product Terms](https://www.microsoft.com/licensing/terms/productoffering/MicrosoftAzureServices/EAEAS)

Reviewer: We generally do not discuss licensing details in Docs. This change will likely not be approved.

```suggestion
Unique to Azure, Azure Hybrid Benefit (AHB) is a licensing benefit that allows you bringing your existing Windows Server and SQL Server licenses with Software Assurance (SA) to Azure. AHB can bring you up to 85% savings compared to pay-as-you-go pricing in Azure SQL, when combined with reservations savings and extended security updates. For more information, see [Microsoft Azure Services Product Terms](https://www.microsoft.com/licensing/terms/productoffering/MicrosoftAzureServices/EAEAS).
```

Author: I did not add any extras about licensing in detail; I just updated options for AHB. Understand rejection. It is better not to say anything than to present only part of the information. How about the link for details (my last sentence) for easy navigation?

#### I want to modernize SQL Server workloads to Azure SQL. What is the next step?

> A great place to start is joining the [Azure Migration and Modernization Program](https://azure.microsoft.com/migration/migration-modernization-program/#overview). When you start a migration project, a good practice is to form a dedicated Migration team to formulate and execute the migration plan. If your company has an assigned Microsoft or Microsoft Partner account team, they can provide guidance regarding the Migration team required skill set and overall process.

Reviewer: ```suggestion
A great place to start is joining the [Azure Migration and Modernization Program](https://azure.microsoft.com/migration/migration-modernization-program/#overview). When you start a migration project, a good practice is to form a dedicated migration team to formulate and execute the migration plan. If your company has an assigned Microsoft or Microsoft Partner account team, they can provide guidance regarding the migration team required skill set and overall process.
```

#### Where can I find migration guides to Azure SQL?

> The following guides help you discover, assess, and migrate from SQL Server to [SQL VM](./virtual-machines/sql-server-to-sql-on-azure-vm-individual-databases-guide.md), [SQL Managed Instance](./managed-instance/sql-server-to-managed-instance-overview.md) and [SQL Database](./database/sql-server-to-sql-database-overview.md). You can consult [Azure Database Migration Guides](/data-migration/) which also contains guides for migrating to another database targets.

Reviewer: In our style guide, "which" always takes a preceding comma, whereas "that" doesn't.

```suggestion
The following guides help you discover, assess, and migrate from SQL Server to [SQL VM](./virtual-machines/sql-server-to-sql-on-azure-vm-individual-databases-guide.md), [SQL Managed Instance](./managed-instance/sql-server-to-managed-instance-overview.md) and [SQL Database](./database/sql-server-to-sql-database-overview.md). You can consult [Azure Database Migration Guides](/data-migration/), which also contains guides for migrating to another database targets.
```
